sample-code/README.md

```diff
@@ -1,26 +1,162 @@
-# Sample Code and E2E Test
-
-![e2e-test](https://github.com/SAP/ai-sdk-js/actions/workflows/e2e-tests.yaml/badge.svg)
+# Sample Code
 
 Sample code to demonstrate the usage of the SAP Cloud SDK for AI.
-Also used as basis for running E2E tests.
+Some of the sample code is also used in E2E test.
+
+### Table of Contents
+
+- [Local Deployment](#local-deployment)
+- [Usage](#usage)
+  - [AI API](#ai-api)
+  - [Foundation Models (Azure OpenAI)](#foundation-models-azure-openai)
+  - [Orchestration](#orchestration)
+  - [Langchain](#langchain)
+
+## Local Deployment
+
+1. Run `pnpm install` to install dependencies.
+2. Create a `.env` file in the `sample-code` directory with your AI core service key by adding the following line:
+
+   ```bash
+   AICORE_SERVICE_KEY='SERVICE_KEY_JSON
+   ```
+
+3. Run `pnpm local` to start the server.
+
+## Usage
+
+### AI API
+
+#### Get all Deployments
+
+GET `/ai-api/deployments`
```

Comment:
[style-guide] https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#use-code-font for HTTP verbs